### PR TITLE
New tile carousel on home page + tiles for community & accessibility

### PR DESCRIPTION
**Pull request by gregwoo-microsoft**
- Adds a reference to the Community Toolkit
- Creates better integration between our various dev tools
- Created a folder for all of the header tiles
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by gregwoo-microsoft** [comment]
FYI @niels9001

File: WinUIGallery/NewControlsPage.xaml [hunk 1]
```diff
@@ -114,10 +114,11 @@
...
-                            <controls:HeaderTile Title="Template Studio" Source="/Assets/template_studio.png" Link="https://marketplace.visualstudio.com/items?itemName=TemplateStudio.TemplateStudioForWinUICs"/>
+                            <controls:HeaderTile Title="Getting Started" Source="/Assets/HomeHeaderTiles/gettingStarted.png" Link="https://docs.microsoft.com/windows/apps/get-started/"/>
+                            <controls:HeaderTile Title="Windows Design" Source="/Assets/HomeHeaderTiles/style-1x.png" Link="https://docs.microsoft.com/windows/apps/design/"/>
+                            <controls:HeaderTile Title="WinUI Repo" Source="/Assets/HomeHeaderTiles/githubTile.png" Link="https://github.com/microsoft/microsoft-ui-xaml"/>
+                            <controls:HeaderTile Title="Template Studio" Source="/Assets/HomeHeaderTiles/template_studio.png" Link="https://marketplace.visualstudio.com/items?itemName=TemplateStudio.TemplateStudioForWinUICs"/>
+                            <controls:HeaderTile Title="Community Toolkit" Source="/Assets/HomeHeaderTiles/toolkit.png" Link="https://apps.microsoft.com/store/detail/windows-community-toolkit-sample-app/9NBLGGH4TLCQ"/>
```
Comment (niels9001) [inline WinUIGallery/NewControlsPage.xaml:121]: You can use this link: https://aka.ms/toolkit/windows

This brings the user directly to the repo, which has a link to the Gallery as well.

**Review by chingucoding — APPROVED** [review]

**Comment by gregwoo-microsoft** [comment]
@niels9001 @chingucoding  let me know what you think, I added a new flipview/carousel functionality + a new accessibility tile

**Comment by chingucoding** [comment]
I think we should recalculate the number of items inside the FlipView based on the width of the app and maybe also use a PipsPager at the bottom? Let me know what you think @gregwoo-microsoft.

**Comment by gregwoo-microsoft** [comment]
Yeah that is a great idea @chingucoding. I can look to add a PipsPager tomorrow. Do you know how to adjust the FlipView based on the app's width?

**Comment by chingucoding** [comment]
I think the best way would be to listen to the FlipViews SizeChanged event and recalculate how many items to show based on that.

**Comment by niels9001** [comment]
@gregwoo-microsoft @chingucoding I've created the following: https://github.com/gregwoo-microsoft/WinUI-Gallery/pull/2

This uses a scrollviewer and will show the scroll buttons whenever the is more content to scroll to. This is also how Store handles its screenshots section and is maybe easier than moving items around in a FlipView😊? (Would be nice to have some sort of adaptive FlipView that can stretch to show more content vs. flipping if not).


Tiles gallery

**Comment by chingucoding** [comment]
That would also work but I think having a PipsPager and FlipView for this would look better even if it is more complicated to set up. Maybe we could even let it switch the view every x seconds?  What are your thoughts @gregwoo-microsoft @niels9001 ?

**Comment by niels9001** [comment]
> That would also work but I think having a PipsPager and FlipView for this would look better even if it is more complicated to set up. Maybe we could even let it switch the view every x seconds? What are your thoughts @gregwoo-microsoft @niels9001 ?

Hmm.. looking at the Store homepage carousel and screenshots gallery on the PDP, they seem to favor a ScrollViewer vs. a FlipView. My worry with a FlipView is that you might end up with e.g. 4 items on the first item and 2 on the second item? Not sure if that'd look great. Also discoverability might not be the greatest as the PipsPager might be missed?

I'd say, in the end this might be an interesting opportunity to recreate something like the Store carousel - this would also allow the landingspage to be more engaging by providing big images with additional information :).

**Comment by chingucoding** [comment]
Alright, lets do that then. Not sure about the big pictures in the future but I think we'll figure that out then.

**Comment by gregwoo-microsoft** [comment]
Love the change @niels9001. Thanks for implementing that. I just merged your PR.

**Comment by niels9001** [comment]
@gregwoo-microsoft Can we get this in?

**Pull request merged**